### PR TITLE
AT-118: Added navigation button to student profile

### PR DESCRIPTION
**Pull request by themegatree**
Adding a link button to the student profile page ( https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-118 )

### CODE REVIEW

File: cypress/integration/students.spec.js [hunk 1]
```diff
@@ -21,4 +21,15 @@ it("Last Name Alpabetical Order", function() {
   cy.visit("/students")
   cy.get("h4:first").should("contain", "PERSON")
 })
+})
+
+describe("Student - Link buttin works", function() {
```
Comment (dearshrewdwit) [inline cypress/integration/students.spec.js:26]: typo: button

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Instead - style the student data as a Card.

Remove the view details button.

When you hover over a student card, make the card outline bolder, and change the background colour - your choice.

When you click on the card, it navigates to the individual student page.

So the whole card should be an anchor link (<a href="url"></a>)

**Comment by themegatree** [comment]
Changes made as requested

File: src/components/Students/StudentList/Student.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+
   return (
-    <div className="container">
+    <div className="studentContainer">
+       <a href={`${url}/${props.student.id}`}>
       <h4><b>Name: {props.student.lastName.toUpperCase()} {props.student.firstName}</b></h4>
       <p>Github Username: {props.student.githubUsername}</p>
       <p>Email: {props.student.email}</p>
+      </a>
+      {/* <li className="btn btn-secondary"><a href={`${url}/${props.student.id}`} id={`button-${props.index}`}>View Profile</a></li> */}
```
Comment (dearshrewdwit) [inline src/components/Students/StudentList/Student.js:16]: Remove comment

**Comment by dearshrewdwit** [comment]
Looks good otherwise

File: src/components/Students/StudentList/Student.js (hunk 2 above)
Comment (themegatree) [reply src/components/Students/StudentList/Student.js:16]: Whoops...

Comment (themegatree) [reply src/components/Students/StudentList/Student.js]: Change has been made

**Review by dearshrewdwit — APPROVED** [review]

**Review by RyanVanDijck — APPROVED** [review]

**Pull request merged**